test/functional/api/v1/searches_controller_test.rb

```diff
@@ -59,7 +59,7 @@ def self.should_respond_to(format)
     end
 
     should "default to first page when page is not a number" do
-      get :show, query: "match", page: "foo", format: :json
+      get :show, query: "match", page: Hash.new, format: :json
```

Comment:
@sonalkr132 I found the tests already written here, I have changed the value of `params[:page]` to a hash instead of a string.
This test should fail prior to the changes made in this PR.